pages/vi/vi-faq.md

```diff
@@ -83,13 +83,13 @@
 
 #### Q14: When I first run BeLL with the "vagrant up" command, why does the download fail?
 
-+ You should first check if [Hashicorp's atlas](https://atlas.hashicorp.com/boxes/search) is up and running by looking at the left sidebar:
++ You should first check if [Hashicorp's atlas](https://status.hashicorp.com) is up and running by looking at the left sidebar:
 
   ![Left Sidebar](images/vi-atlas-status.png)
```

Comment:
Can you also change"[Left Sidebar]" to something like "[Atlas Status]".

Reply:
I have made some changes. please check